content/spaces/kind-quickstart.md

```diff
@@ -76,7 +74,7 @@ export SPACES_TOKEN_PATH="$@/path/to/token.json$@"
 Set the version of Spaces software you want to install.
 
 ```ini
-export SPACES_VERSION=1.1.0
+export SPACES_VERSION=1.2.3
 ```
 
 Set the router host and cluster type. The `SPACES_ROUTER_HOST` is the domain name that's used to access the control plane instances. It's used by the ingress controller to route requests.
```

Comment:
Setting the router host for kind can be trimmed from the quickstart, there's a default value that's assumed so it works without needing to provide it. Same goes for explicitly declaring the `SPACES_CLUSTER_TYPE`.